src/services/PaymentService.ts

```diff
@@ -26,7 +26,15 @@ async function create(
         failUrl,
         cancelUrl,
         metadataId,
-    }: { amount: string; successUrl: string; failUrl: string; cancelUrl: string; metadataId?: string },
+        promotionId,
+    }: {
+        amount: string;
+        successUrl: string;
+        failUrl: string;
+        cancelUrl: string;
+        metadataId?: string;
```

Comment:
@valeriagrazzini Two optional ending arguments seems risky. What if I do submit a promotionId and no metadataId? Perhaps using only 1 last argument `,metadata: { metadataId?:string, promotionId?:string }` works better?